app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencyEvent.kt

```diff
@@ -0,0 +1,6 @@
+package org.simple.clinic.teleconsultlog.medicinefrequency
+
+sealed class MedicineFrequencyEvent
+
+data class SaveMedicineFrequencyClicked(val medicineFrequency: MedicineFrequencies) : MedicineFrequencyEvent()
```

Comment:
Let's add `analyticsName` to all the events we add. You need to extend `UiEvent` interface for `MedicineFrequencyEvent` in order to override the `analyticsName`